src/projetos/CadastroProjetosPage.tsx

```
@@ -36,6 +35,7 @@ import { CustomCheckbox } from '@/components/CustomCheckbox'
 import { NumericInput } from '@/components/NumericInput'
 import { FinalizarProjetoDialog } from '@/components/FinalizarProjetoDialog'
 import { useProjetoContext } from '@/projetos/ProjetoContext'
+import 'react-datepicker/dist/react-datepicker.css';
```

ESLint

[error] Delete `;`

(prettier/prettier)